agent/handlers/types/v2/response.go

```diff
@@ -58,6 +58,7 @@ type ContainerResponse struct {
 	FinishedAt    *time.Time `json:",omitempty"`
 	Type          string
 	Networks      []containermetadata.Network `json:",omitempty"`
+	Health        *api.HealthStatus           `json:",omitempty"`
```

Comment:
`api.HealthStatus`'s  `Status` field is an enum type (`ContainerHealthStatus`) and we should emit a `string` for that instead of a number.

Reply:
It actually implemented the `MarshalJson` [here](https://github.com/aws/amazon-ecs-agent/blob/dev/agent/api/containerstatus.go#L209), so it will appear as `HEALTHY`, `UNHEALTHY` or  `UNKNOWN` in the response.

Reply:
nice! thanks for confirming that.